src/libsync/discoveryphase.h

```diff
@@ -158,6 +158,8 @@ private slots:
 
 public:
     QByteArray _dataFingerprint;
+
+    static const QByteArray DataFingerprintPlaceholder;
```

Comment:
`_dataFingerprintPlaceholder`? We seem to use the same convention than for non static variables. Other option is to turn that into a function of course.